shared/css/fonts/noto-sans.scss

```diff
@@ -36,4 +36,4 @@
  * Thaana/Dhivehi (ދިވެހި): dv-MV
  */
 
-@import url('https://fonts.googleapis.com/css2?family=Noto+Sans+Arabic:wdth,wght@62.5..100,100..900&family=Noto+Sans+Armenian:wdth,wght@62.5..100,100..900&family=Noto+Sans+Bengali:wdth,wght@62.5..100,100..900&family=Noto+Sans+Devanagari:wdth,wght@62.5..100,100..900&family=Noto+Sans+Georgian:wdth,wght@62.5..100,100..900&family=Noto+Sans+Hebrew:wdth,wght@62.5..100,100..900&family=Noto+Sans+JP:wght@100..900&family=Noto+Sans+KR:wght@100..900&family=Noto+Sans+Kannada:wdth,wght@62.5..100,100..900&family=Noto+Sans+Khmer:wdth,wght@62.5..100,100..900&family=Noto+Sans+Myanmar:wght@100;200;300;400;500;600;700;800;900&family=Noto+Sans+SC:wght@100..900&family=Noto+Sans+Sinhala:wdth,wght@62.5..100,100..900&family=Noto+Sans+TC:wght@100..900&family=Noto+Sans+Tamil:wdth,wght@62.5..100,100..900&family=Noto+Sans+Telugu:wdth,wght@62.5..100,100..900&family=Noto+Sans+Thaana:wght@100..900&family=Noto+Sans+Thai:wdth,wght@62.5..100,100..900&family=Noto+Sans:ital,wdth,wght@0,62.5..100,100..900;1,62.5..100,100..900&display=swap');
+@import url("https://fonts.googleapis.com/css2?family=Noto+Sans+Arabic:wdth,wght@62.5..100,100..900&family=Noto+Sans+Armenian:wdth,wght@62.5..100,100..900&family=Noto+Sans+Bengali:wdth,wght@62.5..100,100..900&family=Noto+Sans+Devanagari:wdth,wght@62.5..100,100..900&family=Noto+Sans+Georgian:wdth,wght@62.5..100,100..900&family=Noto+Sans+Hebrew:wdth,wght@62.5..100,100..900&family=Noto+Sans+JP:wght@100..900&family=Noto+Sans+KR:wght@100..900&family=Noto+Sans+Kannada:wdth,wght@62.5..100,100..900&family=Noto+Sans+Khmer:wdth,wght@62.5..100,100..900&family=Noto+Sans+Math&family=Noto+Sans+Myanmar:wght@100;200;300;400;500;600;700;800;900&family=Noto+Sans+SC:wght@100..900&family=Noto+Sans+Sinhala:wdth,wght@62.5..100,100..900&family=Noto+Sans+TC:wght@100..900&family=Noto+Sans+Tamil:wdth,wght@62.5..100,100..900&family=Noto+Sans+Telugu:wdth,wght@62.5..100,100..900&family=Noto+Sans+Thaana:wght@100..900&family=Noto+Sans+Thai:wdth,wght@62.5..100,100..900&family=Noto+Sans:ital,wdth,wght@0,62.5..100,100..900;1,62.5..100,100..900&display=swap");
```

Comment:
@artem-vavilov Do you recall if we need the weight / width for all font families? Noto Sans Math is currently specified without weight and width here and not sure if this is important.

Reply:
FWIW, it looks like Noto Sans Math does not support multiple weights. It only has a regular (400) weight: https://fonts.google.com/noto/specimen/Noto+Sans+Math/tester

Reply:
The only issue is that if a character is only supported by Noto Sans Math, it will be rendered in the regular 400 weight, while the rest of the text will be displayed in the specified weight using the nearest fallback font.

Reply:
I don't think it's important, as Google generates the font query without any attributes for it:
<img src="https://github.com/user-attachments/assets/e170661b-57de-4d80-a767-86b318a69773">

Reply:
And this is how it looks for the regular Noto Sans font:
<img src="https://github.com/user-attachments/assets/3801162c-34b6-4177-a8b2-a95fc1653a3a">

Reply:
The only downside is that if a character is only supported by Noto Sans Math, it will be rendered in the regular 400 weight, while the rest of the text will be displayed in the specified weight using the nearest fallback font.